inc/spbc-settings.php

```diff
@@ -1375,7 +1375,7 @@ function spbc_settings__error__output()
             echo '<div id="spbcTopWarning" class="error" style="position: relative;">'
                  . '<h3 style="display: inline-block;">' . __('Errors:', 'security-malware-firewall') . '</h3>';
             foreach ($errors_out as $value) {
-                echo '<h4>' . $value . '</h4>';
+                echo '<h4>' . spbc_render_links_to_tag($value) . '</h4>';
```

Comment:
maybe we can skip it through some sanitizer, we have cleanUrl there

Reply:
You are right.